src/services/EdFi.DataManagementService.Api/Program.cs

```diff
@@ -6,6 +6,12 @@
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
+app.UsePathBase("/api");
+app.UseRouting();
+
 app.MapGet("/", () => "Data Management Service");
+app.MapGet("/ping", () => Results.Ok(new { CurrentDateTime = DateTime.Now.ToString() }));
```

Comment:
Sorry I didn't mention this yesterday but you should omit the `ToString()`. Even though this is a contrived endpoint and it doesn't really matter, someone might use it as an example. There's no reason to convert our datetimes to strings and it just introduces concerns about default string formats etc. If you remove the `ToString` your test will still behave exactly the same.